packages/frontend/src/components/SafeTranslate.tsx

```diff
@@ -24,9 +24,15 @@ function processDataValue(value) {
 
     // Otherwise we're gonna leave it well enough alone
     return value;
+};
+
+interface Props {
```

Comment:
Generally prefer to place an upper case `I` character before interfaces to help differentiate them from other things. If it hasn't been done in this repo here though, don't worry about it.